docs/PROPSMETHODS.md

```diff
@@ -234,6 +234,14 @@ Refer to the [`react-native` SectionList documentation](https://facebook.github.
 | -------- | -------- |
 | object   | No       |
 
+### `FloatingComponent`
+
+A floating component that will overlay Modalize completely, independent of scrolling. It requires zIndex child with absolute positioning. Check [Flatlist example](../examples).
```

Comment:
Not sure if you wanted to link to the example git repository or page on the documentation.

```suggestion
A floating component inside the modal wrapper that will be independent of scrolling. It requires `zIndex` child with absolute positioning. Check out the [Flatlist example](https://github.com/jeremybarbet/react-native-modalize/blob/master/examples/expo/src/components/modals/FlatList.js).
```